### DISCUSSION TITLE
Possbile docs Improvements

### DISCUSSION THREAD

**Discussion by matmair** [inventree/InvenTree#5957] [User survey]
Where can the docs be approved? What docs are missing/confusing?

**Comment by matmair** [comment]
I think the install instructions could be less fractured and maybe a bit more concise.

**Comment by SchrodingersGat** [comment]
Regarding installation - the docs for the package installer need attention. A number of users have struggled with using the special cli syntax vs normal invoke

**Comment by wolflu05** [comment]
Maybe also the style can be improved to match with the new platform ui theme and be a bit more modern. Comparing them to other OSS docs like gitea the look pretty old in my opinion and the UX on mobile is not that great.

**Comment by BDavid994** [comment]
Hi everyone. I'm a techwriter with a few years of experience and I'm trying to contribute to open source documentation to create a portfolio that I can share publicly. I reached out to @matmair and he was open to the idea of me reviewing the docs and proposing some changes. Note, that I'm no expert in this domain, rather documentation in general.
As a first step, I checked the installation docs and made some initial comments: https://docs.google.com/document/d/15PAz7ugu6JlDYrMNntiRIZL7jmC1fADdJBBJRGAlzAw/edit?usp=sharing
Let me know what you think and if I should carry on with this.

**Reply by matmair** [reply]
Thank you for the notes - they seem like reasonable suggestions. Do I understand correctly that we would end up with 1 general install page, 1 development and 1 for concepts?
Are you comfortable making the suggested changes in the Markdown file itself? It is located here https://github.com/inventree/InvenTree/blob/master/docs/docs/start/intro.md

**Reply by SchrodingersGat** [reply]
@BDavid994 thanks for your input, I am overall in agreement with your suggested changes :)

**Reply by BDavid994** [reply]
Yes, I can create a draft with the changes.

**Comment by matmair** [comment]
Restructuring all the docs to fit this pattern would probably also improve them a lot https://diataxis.fr/
